moVirt/src/main/java/org/ovirt/mobile/movirt/util/message/CreateDialogBroadcastReceiver.java

```diff
@@ -0,0 +1,11 @@
+package org.ovirt.mobile.movirt.util.message;
+
+/**
+ * All activities need to listen for broadcasts Broadcasts.ERROR_MESSAGE and Broadcasts.REST_CA_FAILURE
+ * and call appropriate method in CreateDialogBroadcastReceiverHelper to present dialogs to the user
+ */
+public interface CreateDialogBroadcastReceiver {
```

Comment:
this is just a contract each of our Activities should implement (it is optional, but I think it is clearer this way)